uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkPattern.php

```diff
@@ -82,6 +86,38 @@ public static function del($id)
   }
 
   /**
+   * This method checks if a Work Pattern can be disabled.
+   *
+   * A Work Pattern can be disabled only if it's not the last enabled pattern.
+   *
+   * @param int $id
+   *  The ID of the WorkPattern to be checked
+   *
+   * @throws \CRM_HRLeaveAndAbsences_Exception_InvalidWorkPatternException
+   */
+  private static function checkIfPatternCanBeDisabled($id) {
```

Comment:
since we are inside WorkPattern BAO file I think you can rename this method to **canBeDisabled** instead of this long name.